utils/lib/mix/tasks/add_git_section.ex

```diff
@@ -49,13 +49,22 @@ defmodule Mix.Tasks.Bc.AddGitSection do
     """
   end
 
-  defp commit_snippet_present?(path) do
-    File.read!(path) |> String.contains?("## Commit Your Progress")
+  defp depricated_file?(path) do
+    String.contains?(path, ["DEPRECATED", "DEPRICATED"])
```

Comment:
I think we just need to change the DEPRICATED files to DEPRECATED files and the remove DEPRICATED from this list.